content/millennium/r4/financial/coverage.md

```diff
@@ -234,6 +234,50 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Delete
+
+Delete an existing Encounter-level Coverage.
+
+    DELETE /Coverage/:id
+
+_Implementation Notes_
+
+* Only Encounter-level Coverages may be deleted. Deletes are not currently supported for Patient-level Coverages.
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, system: true) %>
+
+### Headers
+
+<%= headers head: {Authorization: '&lt;OAuth2 Bearer Token>', 'If-Match': 'W/"&lt;Current version of the Coverage resource>"'} %>
+
+### Example
+
+#### Request
+
+    DELETE https://fhir-ehr.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/Coverage/623884-674012
+
+#### Response
+
+<%= headers status: 204 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
```

Comment:
i don't see content-length or content-type coming back from the server

Reply:
removed content-length and content-type. Also updated the screenshot in the PR description https://github.com/cerner/fhir.cerner.com/pull/386/commits/160ac7c1555048d193378a0b5d944e863b8e7988#diff-81b4004b2ae7fd9a288412597630810cL266